draft-ietf-rats-corim.md

```diff
@@ -575,9 +611,6 @@ Described in {{sec-common-validity}}.
 
 * `signer-uri` (index 1): A URI identifying the same organization
 
-* `$$corim-signer-map-extension`: Extension point for future expansion of the
-  Signer map.
-
```

Comment:
Why are we removing extensibility? For example, it will be easier to add vLEI and whatever WIMSE comes up with in the future.